v3/setup/splunk.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# SPLUNK_FORWARD_SERVER shoudl be delcared as a config in the setup-credentials util
+
+source /etc/environment
+
+etcdctl set /splunk/SPLUNK_FORWARD_SERVER $SPLUNK_FORWARD_SERVER
```

Comment:
I was thinking that @ts3ng could add it via the `secrets.json` with the cluster. Have it in Dynamo and have the setup-credentials script write it to `/etc/environment`. Or do I not understand how that part works? 😕